tests/system/action/user/test_update.py

```diff
@@ -400,121 +392,350 @@ def test_update_permission_OML_not_high_enough(self) -> None:
             "user.update",
             {
                 "id": 111,
-                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_ORGANISATION,
+                "group_$_ids": {1: [1]},
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/111",
+            {"group_$_ids": ["1"], "group_$1_ids": [1]},
+        )
+
+    def test_update_permission_group_C_committee_manager(self) -> None:
+        """ May update group C group_$_ids by committee permission """
+        self.permission_setup()
+        self.update_model(
+            f"user/{self.user_id}",
+            {
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$_management_level": ["60"],
```

Comment:
Same here: I would expect it to fail since the request user is not part of the committee

Reply:
now with committee_ids